scripts/lib.sh

```diff
@@ -105,31 +105,52 @@ install-py-dev() {
     upgrade-pip
     # It places us within top src dir to be at the same path as setup.py
     # This helps when getsentry calls into this script
-    cd "${HERE}/.." || exit
+    cd "${HERE}/.." :|| exit
+
     echo "--> Installing Sentry (for development)"
     if query-apple-m1; then
         # This installs pyscopg-binary2 since there's no arm64 wheel
         # This saves having to install postgresql on the Developer's machine + using flags
         # https://github.com/psycopg/psycopg2/issues/1286
         pip install https://storage.googleapis.com/python-arm64-wheels/psycopg2_binary-2.8.6-cp38-cp38-macosx_11_0_arm64.whl
-        # This install confluent-kafka from our GC storage since there's no arm64 wheel
-        # https://github.com/confluentinc/confluent-kafka-python/issues/1190
-        pip install https://storage.googleapis.com/python-arm64-wheels/confluent_kafka-1.5.0-cp38-cp38-macosx_11_0_arm64.whl
-        # uwsgi does not properly install via pyenv Python installations
-        # https://github.com/unbit/uwsgi/issues/2361
-        LDFLAGS=-L/opt/homebrew/Cellar/gettext/0.21/lib  pip install "uWSGI==2.0.19.1"
     fi
+
+    # SENTRY_LIGHT_BUILD=1 disables webpacking during setup.py.
+    # Webpacked assets are only necessary for devserver (which does it lazily anyways)
+    # and acceptance tests, which webpack automatically if run.
+    SENTRY_LIGHT_BUILD=1 pip install -e '.[dev]'
```

Comment:
I'm only moving the `fx_profile` block after this. It only makes sense to patch *after* the package has been installed. See #29887 for original work.